packages/suite/src/components/wallet/CoinmarketTag/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import { variables } from '@trezor/components';
+
+const TagRow = styled.div`
+    display: flex;
+`;
+
+const Tag = styled.div`
+    margin: 0 16px 2px 16px;
+    padding: 0 8px;
+    border-radius: 8px;
+    background: ${props => props.theme.TYPE_ORANGE};
+    color: ${props => props.theme.TYPE_WHITE};
+    font-size: ${variables.FONT_SIZE.TINY};
+    line-height: 21px;
+    text-transform: uppercase;
+`;
+
+interface Props {
+    tag?: string;
+}
+
+const CoinmarketTag = ({ tag }: Props) => <TagRow>{tag && <Tag>{tag}</Tag>}</TagRow>;
+
+export default CoinmarketTag;
```

Comment:
Props ⇒ ComponentProps 
one more nitpick - please start prefixing Props type by Component name. No big deal but it’s more explicit, easier to export type later, nicer error messages...

```suggestion
interface CoinmarketTagProps {
    tag?: string;
}

export const CoinmarketTag = ({ tag }: CoinmarketTagProps) => <TagRow>{tag && <Tag>{tag}</Tag>}</TagRow>;
```